runtime/queries/adl/highlights.scm

```diff
@@ -0,0 +1,39 @@
+; adl
+
+[
+"module"
+"struct"
+"union"
+"type"
+"newtype"
+"annotation"
+] @keyword
+
+(scoped_name) @variable
+(comment) @comment
+(doc_comment) @info
+(name) @type
+
+(ERROR) @error
```

Comment:
This pattern can be removed. Eventually we want to add a per-language configuration for this so that you could customize whether you see error highlighting. It's usually very noisy so by default we should remove the pattern